src/DynamoCore/Models/DynamoModelCommands.cs

```diff
@@ -40,6 +40,7 @@ public void ExecuteCommand(RecordableCommand command)
         }
         
         private PortModel activeStartPort;
+        private PortModel[] activeStartPorts;
```

Comment:
I was wondering if we can remove `activeStartPort` and use `activeStartPorts` even for single connection.